demo/app/page.tsx

```diff
@@ -8,6 +8,10 @@ import { Staking } from "@figmentio/elements";
 const options = [
   { label: "ETH", value: "staking-eth" },
   { label: "BTC (Babylon)", value: "staking-btc" },
+  // {
+  //   label: "BTC (Babylon) - Custom Wallet (Tomo)",
+  //   value: "staking-btc-custom",
+  // },
```

Comment:
Keeping this for testing reasons, but it's not functional unless you manually connect Tomo wallet so it's not super useful to external users.